lib/linguist/languages.yml

```diff
@@ -3738,12 +3738,12 @@ Ruby:
   - Deliverfile
   - Fastfile
   - Gemfile
-  - Gemfile.lock
```

Comment:
`Gemfile.lock` isn't it's own format, that's an incorrect characterization. The lockfile is a generated file type that's an artifact of the `Gemfile` which is definitely Ruby (or at least it can be). It doesn't make any sense at all to classify these two independently from each other.

I'll 👍 the rest of this PR if you pull out this change.